### PR TITLE
Wire the app detail page

### PR DESCRIPTION
**Pull request by oponder**
Wire up the app detail page to real data.

Screenshot 2021-02-09 at 9 21 03 PM
Screenshot 2021-02-09 at 9 21 17 PM
Screenshot 2021-02-09 at 9 20 48 PM

### CODE REVIEW

File: src/components/UI/Display/Apps/AppDetailNew/AppDetailPage.tsx [hunk 1]
```diff
@@ -163,48 +192,62 @@ const AppDetail: React.FC<IAppDetailPageProps> = (props) => {
               Install in Cluster
             </Button>
           </Upper>
-          <Lower>{props.catalogName}</Lower>
+          <Lower>
+            <CatalogLabel
+              catalogName={props.catalogName}
+              // isManaged={props.catalogIsManaged}
```
Comment (axbarsan) [inline src/components/UI/Display/Apps/AppDetailNew/AppDetailPage.tsx:198]: Is this on purpose?

File: src/components/Apps/AppDetail/AppDetail.tsx [hunk 2]
```diff
@@ -43,18 +51,23 @@ const AppDetail: React.FC = () => {
...
+          appTitle={app.name}
+          appIconURL={app.icon}
+          catalogName={catalog.spec.title}
+          catalogIcon={catalog.spec.logoURL}
+          chartVersion={app.version}
+          createDate={app.created}
+          includesVersion={app.appVersion}
+          description={app.description}
+          website={app.home}
+          keywords={app.keywords || []}
```
Comment (axbarsan) [inline src/components/Apps/AppDetail/AppDetail.tsx:65]: Can we handle this fallback at the `AppDetailPage` level (using `defaultProps`)?

**Review by axbarsan — APPROVED** [review]
LGTM in general

File: src/components/UI/Display/Apps/AppDetailNew/AppDetailPage.tsx (hunk 1 above)
Comment (oponder) [reply src/components/UI/Display/Apps/AppDetailNew/AppDetailPage.tsx:198]: Nope! I'll clean that up.

File: src/components/UI/Display/Apps/AppDetailNew/AppDetailPage.tsx [hunk 3]
```diff
@@ -261,7 +260,7 @@ const AppDetail: React.FC<IAppDetailPageProps> = (props) => {
             {props.website}
           </Detail>
 
-          {props.keywords.length > 0 && (
+          {props.keywords && props.keywords.length > 0 && (
```
Comment (axbarsan) [inline src/components/UI/Display/Apps/AppDetailNew/AppDetailPage.tsx:263]:
```suggestion
          {props.keywords!.length > 0 && (
```

Comment (oponder) [reply src/components/UI/Display/Apps/AppDetailNew/AppDetailPage.tsx:263]: fancy!

**Pull request merged**